submissions/iakow/html-css-popup/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css">
+  <title>Hooli</title>
+</head>
+
+<body>
+  <nav>
+    <ul>
+      <li title="E-mail">
+        <a class="nav-item email" href="#"></a>
+      </li>
+
+      <li title="Images">
+        <a class="nav-item images" href="#"></a>
+      </li>
+
+      <li class="relative-box" title="Applications">
+        <input class="popup-ctrl" id="popup-ctrl" type="checkbox">
+
+        <label class="nav-item apps" for="popup-ctrl"></label>
```

Comment:
Why do you need two labels here?

Reply:
To be able to close the popup by clicking anywhere else in the document.

Reply:
Got it! Ok!